components/products/ImageGallery.tsx

```
@@ -0,0 +1,72 @@
+              onClick={() => setSelectedImageIndex(index)}
+              className={cn(
+                "relative aspect-square overflow-hidden rounded-md bg-muted transition-all duration-200",
+                "hover:ring-2 hover:ring-salsa-500 hover:ring-offset-2",
+                "focus:outline-none focus:ring-2 focus:ring-salsa-500 focus:ring-offset-2",
+                selectedImageIndex === index
+                  ? "ring-2 ring-salsa-500 ring-offset-2 opacity-100"
+                  : "opacity-60 hover:opacity-100"
+              )}
+              aria-label={`View image ${index + 1} of ${galleryImages.length}`}
+              aria-pressed={selectedImageIndex === index}
+            >
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)